### ISSUE TITLE
'File' is ambiguous in the namespace 'TagLib'.

### ISSUE THREAD

**Issue by carrzkiss** [mono/taglib-sharp#203]
In my vb.net web site.
I have the following.

```
Namespace TagLib
End Namespace
```

Within this Namespace, I have this.

`Dim mp3 As File = File.Create("~/FileUpload/" + getfile & ".mp3")`

I get the error message when the site runs.
'File' is ambiguous in the namespace 'TagLib.'

Any information on correcting this so that it will work will be much appreciated.

Wayne

**Comment by tcHylmrX** [comment]
Your Namespace is named the very same as the File object you are using. Make your Namespace a little different.

**Comment by carrzkiss** [comment]
I renamed the namespace to

> Namespace Upload

And I still get 

> Error	BC30560	'File' is ambiguous in the namespace 'TagLib'.	Read

I'm at a loss.
This all used to work until I had to make the Virtual Directory an Application within IIS.
And now, it does not.
I mean, the warning has always been there, and it would run without issue.
But since it is an Application now, it is giving me this error.

Since I changed the Namespace, it is still giving the error.
How would I write the code to make the MP3, a part of TagLib?
I tried this and it still gives the same error.
`Dim MP3 As TagLib.File`

**Comment by carrzkiss** [comment]
Figured it out.
In my web.config file, it was referencing the wrong version number of the TagLib-Sharp.dll 2.1.0.0
`<add assembly="taglib-sharp, Version=2.1.0.0, Culture=neutral, PublicKeyToken=MyToken"/>`
I have the newer one, TagLibsharp.dll 2.2.0.0
`<add assembly="taglibsharp, Version=2.2.0.0, Culture=neutral, PublicKeyToken=MyToken"/>`

Once I corrected the version number and the name within the web.config file.
And rebuild the project, the errors went away.

Have a good one.
